src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/CsvFormatResponseIT.java

```diff
@@ -142,6 +156,460 @@ public void fieldOrderWithScriptFields() throws IOException {
         verifyFieldOrder(expectedFields, query);
     }
 
+    //region Tests migrated from CSVResultsExtractorTests
+
+    @Test
+    public void simpleSearchResultNotNestedNotFlatNoAggs() throws Exception {
+        String query = String.format(Locale.ROOT, "select dog_name,age from %s/dog order by age", TEST_INDEX_DOG);
+        final CSVResult csvResult = executeCsvRequest(query, false);
+
+        List<String> headers = csvResult.getHeaders();
+        Assert.assertEquals(2, headers.size());
+        Assert.assertTrue("name should be on headers", headers.contains("dog_name"));
+        Assert.assertTrue("age should be on headers", headers.contains("age"));
+
+        List<String> lines = csvResult.getLines();
+        Assert.assertEquals(2, lines.size());
+        Assert.assertTrue("rex,2".equals(lines.get(0)) || "2,rex".equals(lines.get(0)));
```

Comment:
I wasn't aware of this. So we may generate columns in random order?

Reply:
I think we fixed that bug, but I don't want to run the queries one by one and update the code for those. I know it makes sense, but this is more a migration work, not improving the tests (though I am doing some of that), so if you want we can create an issue and track that separately.

Reply:
Sounds good!